storm-core/src/clj/org/apache/storm/config.clj

```diff
@@ -41,11 +41,30 @@
   (dofor [f (seq (.getFields Config))]
          (.get f nil)))
 
-
+;; TODO this function and its callings will be replace when nimbus and supervisor move to Java
 (defn cluster-mode
   [conf & args]
   (keyword (conf STORM-CLUSTER-MODE)))
 
+(defn sampling-rate
+  [conf]
+  (->> (conf TOPOLOGY-STATS-SAMPLE-RATE)
+    (/ 1)
+    int))
+
+;; TODO this function together with sampling-rate are to be replaced with Java version when util.clj is in
+(defn mk-stats-sampler
+  [conf]
+  (even-sampler (sampling-rate conf)))
+
+;; TODO this function is to be replaced with Java version when util.clj is in
+(defn read-default-config
+  []
+  (clojurify-structure (Utils/readDefaultConfig)))
```

Comment:
These 2 functions are actually depending on util.clj. I may just translate those two util functions within this pull request and delete these two.